tests/software/rasusa/main.nf

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { RASUSA } from '../../../software/rasusa/main.nf' addParams( options: [:])
```

Comment:
```suggestion
include { RASUSA } from '../../../software/rasusa/main.nf' addParams( options: ['suffix':'100X'])
```

Reply:
This is where you can specify `options.suffix` @thanhleviet as I have done above.

Reply:
Thanks, I guess I'm getting used to the logic. Code updated.

Reply:
Awesome!